doc/documentation.tex

```diff
@@ -396,7 +404,14 @@ \subsubsection{Totani Format}
 Finally, a linear interpolation in time and log cubic spline interpolation in energy are used to determine the spectral number luminosity at an arbitrary time and energy.%
 \footnote{This approach closely follows the one used in code provided by Totani. There is excellent agreement of the calculated fluxes between Totani’s code and sntools, with differences of at most a few per mille due to slight differences in the numerical interpolation algorithms used.}
 
+\subsection{Pre Supernova Time Binning}\label{sec:pre-supernova-time-binning}
+As mentioned in section~\ref{sec:more-details}, when using pre supernova models, the size of the time bins used for calculations within SNTools can be specified through the optional argument \texttt{--binsize <value>}.
+During the development of support for pre supernova models, a study was performed to optimse the time bin size for Hyper-Kamiokande. The study concluded that a bin size of \SI{1}{s} was the best choice,  
+limiting any artificial step features in the distribution of events arising from the nature of the time binning. As a result, \SI{1}{s} is chosen as the default value of this argument.
 
+The study as a whole suggested an absolute lower limit on the bin size of \SI{50}{ms}, arising from statistical considerations, and an upper limit of 300s to preserve the important features in the event distribution in time before core collapse, 
+such as the accelerated event rate close to the core collapse of the star. 
```

Comment:
The 50 ms and 300 s are HK-specific; exact numbers will likely be different for a different detector with other statistics. I’d therefore suggest removing them here.

Will NuPhys have proceedings? If so, we could refer to your proceedings here, so people can look up more details of how this time bin optimisation works.